src/assembly/agent-load-test/agent-load-test.bat

```diff
@@ -0,0 +1,113 @@
+@REM ----------------------------------------------------------------------------
+@REM  Copyright 2001-2006 The Apache Software Foundation.
```

Comment:
Would it be possible to remove such duplication and to require Cygwin if we want to keep low-level scripting?